src/main/java/com/stripe/model/Dispute.java

```diff
@@ -128,6 +128,13 @@ public class Dispute extends ApiResource
   @SerializedName("status")
   String status;
 
+  /**
+   * The enum that describes the dispute loss outcome. If the dispute is not lost, this field will
+   * be absent.
+   */
+  @SerializedName("loss_reason")
+  String lossReason;
```

Comment:
Might actually be nice to point to the documentation for this field on Stripe.
We could also have considered doing a merge master of the original source but
I don't see if in the latest master of their lib. https://github.com/stripe/stripe-java/blob/master/src/main/java/com/stripe/model/Dispute.java